File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
import java.util.*;

class Solution {
    public int solution(String[] babbling) {
        String[] words = {"aya", "ye", "woo", "ma"};
        int count = 0;
        for (String b : babbling) {
            String temp = b;
            for (String w : words) {
                temp = temp.replace(w, " ");
            }
            if (temp.trim().isEmpty()) {
                count++;
            }
        }
        return count;
    }
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    const words = ["aya", "ye", "woo", "ma"];
    return babbling.filter(b => {
        let temp = b;
        for (let w of words) {
            temp = temp.replaceAll(w, " ");
        }
        return temp.trim() === "";
    }).length;
}
